### DISCUSSION TITLE
IsSelectAllTextOnFocus on Input Focus

### DISCUSSION THREAD

**Discussion by gmondin** [dotnetcore/BootstrapBlazor#3577] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hi, some versions ago (I'm using .net 7, bootstrapBlazor.7.1.12) the function IsSelectAllTextOnFocus stop working when I put the focus on the input, using FocusAsync, its only work at the first time after rendering, I was looking at the code and I saw that the invoke of the method is only after rendering, its possible be back for working after use the focusAsync function? if is possible, can I have a release on .net 7 version?
thank you very much

### .NET Version

.NET 7

**Comment by bb-auto** [comment]
@gmondin Thank you for reporting. We will give triage later.

**Comment by densen2014** [comment]
please try the last version, it's works in net7 too

**Comment by gmondin** [comment]
Hi, I tried, but didn't work.
My code is like this:
Component:
   <BootstrapInputNumber @bind-Value="@_qta" IsSelectAllTextOnFocus="true" ShowLabel="false" @ref="_txtQta" OnEnterAsync="OnQtaEnterAsync" />
   
   code:
   
       await _txtQta.FocusAsync();

if I use this, the focus works, but, this part doesn't: SelectAllTextOnFocus="true".

**Comment by gmondin** [comment]
for more information, if I use this: await _txtQta.SelectAllTextAsync(); works, but all the things was build with focusAsync and IsSelectAllTextOnFocus="true", so, if is possible, is better on the old way.
thanks

**Comment by ArgoZhang** [comment]
@gmondin  please provider mini-repro-project. Please do not paste incomplete code snippets, it will not help much in solving the problem
